packages/pharos/src/components/sheet/pharos-sheet.ts

```diff
@@ -0,0 +1,367 @@
+import { PharosElement } from '../base/pharos-element';
+import { html, nothing } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import type { TemplateResult, CSSResultArray, PropertyValues } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { sheetStyles } from './pharos-sheet.css';
+import focusable from '../../utils/focusable';
+
+import ScopedRegistryMixin from '../../utils/mixins/scoped-registry';
+import { PharosButton } from '../button/pharos-button';
+import { PharosHeading } from '../heading/pharos-heading';
+import { FocusTrap } from '@ithaka/focus-trap';
+
+const CLOSE_BUTTONS = `[data-sheet-close],[data-pharos-component="PharosButton"]#close-button`;
+const FOCUS_ELEMENT = `[data-sheet-focus]`;
+
+/**
+ * Pharos sheet component.
+ *
+ * @tag pharos-sheet
+ *
+ * @slot description - Content that describes the primary message or purpose of the sheet.
+ * @slot - Contains the content of the sheet body.
+ * @slot footer - Contains the content of the sheet footer.
+ *
+ * @fires pharos-sheet-open - Fires when the sheet is about to open - cancelable
+ * @fires pharos-sheet-opened - Fires when the sheet has opened
+ * @fires pharos-sheet-close - Fires when the sheet is about to close - cancelable
+ * @fires pharos-sheet-closed - Fires when the sheet has closed
+ *
+ */
+export class PharosSheet extends ScopedRegistryMixin(PharosElement) {
+  static elementDefinitions = {
+    'pharos-button': PharosButton,
+    'pharos-heading': PharosHeading,
+    'focus-trap': FocusTrap,
+  };
+
+  /**
+   * Indicates if the sheet is open.
+   * @attr open
+   */
+  @property({ type: Boolean, reflect: true })
+  public open = false;
+
+  /**
+   * Indicates if the sheet is expanded.
+   * @attr expanded
+   */
+  @property({ type: Boolean, reflect: true })
+  public expanded = false;
+
+  /**
+   * Indicates if the sheet is allowed to expand.
+   * @attr enbaleExpansion
+   */
+  @property({ type: Boolean, reflect: true })
+  public enbaleExpansion = true;
+
+  /**
+   * Indicates if the sheet contains close button.
+   * @attr hasClose
+   */
+  @property({ type: Boolean, reflect: true })
+  public hasClose = true;
+
+  /**
+   * Text content for the sheet header
+   * @attr header
+   */
+  @property({ type: String, reflect: true })
+  public header = 'Sheet header';
+
+  /**
+   * Indicates if the sheet footer should contain a divider.
+   * @attr footer-divider
+   */
+  @property({ type: Boolean, reflect: true, attribute: 'footer-divider' })
+  public footerDivider = false;
```

Comment:
is this used?

Reply:
Probably not. I think this was leftover from copy and pasting the modal component as a starting point

Reply:
Yeah, it is used in css only, to determine if we want to show a border on the footer